cdist/config.py

```diff
@@ -176,8 +177,15 @@ def commandline(cls, args):
                               " ".join(failed_hosts))
 
     @classmethod
-    def _resolve_remote_cmds(cls, args, host_base_path):
-        control_path = os.path.join(host_base_path, "ssh-control-path")
+    def _resolve_ssh_control_path(cls):
+        base_path = tempfile.mkdtemp()
+        control_path = os.path.join(base_path, "s")
```

Comment:
Won't this create a new control_path on every call? Shouldn't we store it in the Config object as it is valid for the whole run?